umpost/um2netcdf.py

```diff
@@ -965,8 +965,26 @@ def convert_32_bit(cube):
 
 
 def fix_time_coord(cube, verbose):
+    """
+    Ensures cube has a 'time' coordinate dimension.
+
+    Coordinate dimensions are reordered to ensure 'time' is the first dimension.
+    Cubes are modified in place, although it is possible iris will return new
+    copies of cubes. UM ancillary files are ignored.
```

Comment:
Just to be 100% accurate, it might be worth changing it to "UM ancillary files with no time dimension are ignored", as it is possible for ancillary files to not have any time dimension

Reply:
OK, it's in dff0b7d7.